airflow/utils/db.py

```diff
@@ -1282,6 +1287,7 @@ def _check_migration_errors(session: Session = NEW_SESSION) -> Iterable[str]:
         check_bad_references,
     )
     for check_fn in check_functions:
+        log.info("running pre-migration check %s", check_fn.__name__)
```

Comment:
This is the one I'm not sure about - if the check is a no-op (which it should be for most of the time once it's run once?) then isn't this just noise?

Reply:
ok -- removed that one @ashb